target/linux/brcm47xx/image/Makefile

```diff
@@ -822,6 +822,15 @@ define Device/netgear-wgr614-v10-na
 endef
 TARGET_DEVICES += netgear-wgr614-v10-na
 
+define Device/netgear-r6200-v1
+  DEVICE_TITLE := Netgear R6200 V1
```

Comment:
Please split up DEVICE_TITLE:
```
DEVICE_VENDOR := Netgear
DEVICE_MODEL := R6200
DEVICE_VARIANT := V1
```
See https://github.com/openwrt/openwrt/pull/2124 - thanks!

Reply:
ah, I forgot that it is NETGEAR (upper case). The other tickets are already doing it. e.g. https://github.com/openwrt/openwrt/pull/2222.

Reply:
Thanks, I fixed it in my latest push. I didn't notice it in the top of that thread earlier